docs/static/security/es-security.asciidoc

```diff
@@ -87,7 +87,7 @@ Use the <<plugins-outputs-elasticsearch,`elasticsearch output`'s>> <<plugins-out
 output {
   elasticsearch {
     hosts => ["https://...] <1>
-    cacert => '/etc/logstash/config/certs/ca.crt' <2>
+    ssl_certificate_authorities => '/etc/logstash/config/certs/ca.crt' <2>
```

Comment:
```suggestion
    ssl_certificate_authorities => ['/etc/logstash/config/certs/ca.crt'] <2>
```
After looking at more examples in docs I think it might be best to have this explicitly in list format.